uploadNeuroDB/NeuroDB/objectBroker/ConfigOB.pm

```diff
@@ -348,6 +352,66 @@ sub getLivingPhantomRegex {
     return &$getConfigSettingRef($self, LIVING_PHANTOM_REGEX);
 }
 
+=head3 getCreateNii()
+
+Get the create_nii Config setting.
+
+RETURN: (boolean) 1 if create_nii is set to Yes in the Config module, 0 otherwise
+
+=cut
+sub getCreateNii {
```

Comment:
I think it would be a good idea to have a private function `getBoolean` that the other methods would call instead of "hardcoding" the logic inside each method. What do you think?

Reply:
I agree!